pennylane/ops/qutrit/channel.py

```diff
@@ -0,0 +1,223 @@
+
+    .. math::
+        K_0 = K_{0,0} = \sqrt{1-p} \begin{bmatrix}
+                1 & 0 & 0\\
+                0 & 1 & 0\\
+                0 & 0 & 1
+                \end{bmatrix}, \quad
+        K_{i,j} = \sqrt{\frac{p}{8}}X^iZ^j
+
+    Where:
+
+    .. math::
+        X = \begin{bmatrix}
+                0 & 1 & 0 \\
+                0 & 0 & 1 \\
+                1 & 0 & 0
+                \end{bmatrix}, \quad
+        Z = \begin{bmatrix}
+                1 & 0 & 0\\
+                0 & \omega & 0\\
+                0 & 0 & \omega^2
+                \end{bmatrix}
+
+
+    These relations create the following Kraus matrices:
+
+    .. math::
+        \begin{matrix}
+            K_0 = K_{0,0} = \sqrt{1-p} \begin{bmatrix}
+                1 & 0 & 0\\
+                0 & 1 & 0\\
+                0 & 0 & 1
+                \end{bmatrix}&
+            K_1 = K_{0,1} = \sqrt{\frac{p}{8}}\begin{bmatrix}
+                1 & 0 & 0\\
+                0 & \omega & 0\\
+                0 & 0 & \omega^2
+                \end{bmatrix}&
+            K_2 = K_{0,2} = \sqrt{\frac{p}{8}}\begin{bmatrix}
+                1 & 0 & 0\\
+                0 & \omega^2 & 0\\
+                0 & 0 & \omega
+                \end{bmatrix}\\
+            K_3 = K_{1,0} = \sqrt{\frac{p}{8}}\begin{bmatrix}
+                0 & 1 & 0 \\
+                0 & 0 & 1 \\
+                1 & 0 & 0
+                \end{bmatrix}&
+            K_4 = K_{1,1} = \sqrt{\frac{p}{8}}\begin{bmatrix}
+                0 & \omega & 0 \\
+                0 & 0 & \omega^2 \\
+                1 & 0 & 0
+                \end{bmatrix}&
+            K_5 = K_{1,2} = \sqrt{\frac{p}{8}}\begin{bmatrix}
+                0 & \omega^2 & 0 \\
+                0 & 0 & \omega \\
+                1 & 0 & 0
+                \end{bmatrix}\\
+            K_6 = K_{2,0} = \sqrt{\frac{p}{8}}\begin{bmatrix}
+                0 & 0 & 1 \\
+                1 & 0 & 0 \\
+                0 & 1 & 0
+                \end{bmatrix}&
+            K_7 = K_{2,1} = \sqrt{\frac{p}{8}}\begin{bmatrix}
+                0 & 0 & \omega^2 \\
+                1 & 0 & 0 \\
+                0 & \omega & 0
+                \end{bmatrix}&
+            K_8 = K_{2,2} = \sqrt{\frac{p}{8}}\begin{bmatrix}
+                0 & 0 & \omega \\
+                1 & 0 & 0 \\
+                0 & \omega^2 & 0
+                \end{bmatrix}
+        \end{matrix}
+
+    Where :math:`\omega=\exp(\frac{2\pi}{3})`  is the third root of unity.
+    where :math:`p \in [0, 1]` is the depolarization probability and is equally
+    divided in the application of all qutrit Pauli operators.
+
+    .. note::
+
+        The Kraus operators :math:`\{K_0 \ldots K_8\}` used are the representations of the single qutrit Pauli group.
+        These Pauli group operators are defined in [`1 <https://arxiv.org/pdf/quant-ph/9802007>`_] (Eq. 5).
+        The Kraus Matrices we use are adapted from [`2 <https://doi.org/10.48550/arXiv.1905.10481>`_] (Eq. 5).
```

Comment:
The first link goes directly to a PDF, and the second link goes to the arxiv webpage and then the user has to open the PDF themselves. I'm not opposed to using either of those links, but could you update them so that they both either open the PDF or arxiv?

Reply:
Good catch - suggest linking to the webpage and not the PDF. PDFs sometimes autodownload in people's browsers and I'm not keen on that (drives me nuts when it happens to me!)

Reply:
```suggestion
        These Pauli group operators are defined in [`1 <https://doi.org/10.48550/arXiv.quant-ph/9802007>`_] (Eq. 5).
        The Kraus Matrices we use are adapted from [`2 <https://doi.org/10.48550/arXiv.1905.10481>`_] (Eq. 5).
```